src/transaction/log_replication.cpp

```diff
@@ -104,16 +102,15 @@ namespace cublog
     , m_perfmon_redo_sync { PSTAT_REDO_REPL_LOG_REDO_SYNC }
     , m_perf_stat_idle { cublog::perf_stats::do_not_record_t {} }
   {
-    log_zip_realloc_if_needed (m_undo_unzip, LOGAREA_SIZE);
-    log_zip_realloc_if_needed (m_redo_unzip, LOGAREA_SIZE);
+    log_zip_realloc_if_needed (m_redo_context.m_undo_zip, LOGAREA_SIZE);
```

Comment:
Remove lines after including function calls in context ctor.

Reply:
Done